csharp/src/Apache.Arrow/Arrays/ArrayData.cs

```diff
@@ -22,6 +22,8 @@ namespace Apache.Arrow
 {
     public sealed class ArrayData : IDisposable
     {
+        public static readonly int RecalculateNullCount = -1;
```

Comment:
Can we leave this `private` for now? We can always make it public later.

Reply:
Changed